test/rulesets/Base/evalpoly.jl

```diff
@@ -22,14 +22,14 @@
             sp = (np..., np...)
             x, ẋ, x̄ = randn(T, sx...), randn(T, sx...), randn(T, sx...)
             p = [randn(T, sp...) for _ in 1:m]
+            @show p
             ṗ = [randn(T, sp...) for _ in 1:m]
             p̄ = [randn(T, sp...) for _ in 1:m]
             Ω = evalpoly(x, p)
             Ω̄ = randn(T, size(Ω)...)
             frule_test(evalpoly, (x, ẋ), (p, ṗ))
-            frule_test(evalpoly, (x, ẋ), (Tuple(p), Tuple(ṗ)))
+            frule_test(evalpoly, (x, ẋ), (Tuple(p), rand_tangent(Tuple(p))))
```

Comment:
Would it make sense to use `rand_tangent` for all tangents in this test set?

Reply:
It would make sense to use `rand_tangent` in a whole ton of places.
Infact, we should use it by default everywhere.
See: https://github.com/JuliaDiff/ChainRulesTestUtils.jl/issues/67

I am not inclines to get through and change to using it manually in a bunch of places when what we should do is change to using it automatically everywhere.

I made the change here as it was shorted than expressly writing out the Composite
I could make it in more places here, but that would be longer than what we have.
so I figure the rest could wait for us to do tangent generation automatically

Reply:
Okay, makes sense